controllers/storageclaim_controller.go

```diff
@@ -49,29 +50,59 @@ import (
 )
 
 const (
-	storageClassClaimFinalizer  = "storageclassclaim.ocs.openshift.io"
-	storageClassClaimAnnotation = "ocs.openshift.io/storagesclassclaim"
+	storageClaimFinalizer  = "storageclaim.ocs.openshift.io"
+	storageClaimAnnotation = "ocs.openshift.io/storageclaim"
+
+	pvClusterIDIndexName = "index:pvClusterID"
+	vscDriverIndexName   = "index:vscCSIDriver"
```

Comment:
it is not obvious what `pv` and `vsc` stands for in this context. 
can we have more descriptive names? at least in the value

```suggestion
	pvClusterIDIndexName = "index:PresistantVolumeClusterID"
	vscDriverIndexName   = "index:VolumeSnapshotContentCSIDriver"
```

Reply:
I thought in kube world `pv` is fairly know and so does `vsc` when clubbed w/ `CSI`. No qualms w/ the suggestion.

Reply:
done.